doc/src/_static/custom.css

```diff
@@ -0,0 +1,116 @@
+/* Completely disable blockquotes. The Furo defaults make block quotes very*/
+/* apparent. However, the parameter blocks from numpydoc render as block
+ * quotes, which looks bad. Furthermore, it is very easy for things to
+ * accidentally have block quotes due to the fact that RST makes indented
+ * blocks render as a block quote (unlike Markdown which makes indented blocks
+ * render as code). There aren't any places in the documentation that use a
```

Comment:
Well, this is your documentation, so do whateverr you want -- I'll note that I made blockquotes apparent *because* it was too easy to inject them by accident with reST, making them unusable when you *actually* want them. :)

Reply:
I could make an effort to remove all the blockquotes, and maybe I should. This is a much easier thing for now. A big problem is that numpydoc is using blockquote for the parameters, which makes them look pretty bad. So we'd need to fix that (we're actually currently using a forked version of numpydoc, which could be part of the problem, but that just highlights the complexity). SymPy has a lot of documentation (over 270 RST files, most of which are pulling in documentation from docstrings). So any kind of content clean up like this would take a bit of effort, which is more than what I want to do for this PR.